client_connection/manager.go

```diff
@@ -100,6 +102,10 @@ func statusNotConnected() ConnectionStatus {
 	return ConnectionStatus{NotConnected, "", nil}
 }
 
+func statusDisconnecting() ConnectionStatus {
```

Comment:
Strange to nullify sessions ID, while session still exists

Reply:
Does session have any meaning on disconnecting state?

Reply:
Probably we dont have real case